docs/how_to/cluster_hard_way.md

```diff
@@ -50,20 +50,7 @@ M3DB_HOST_ID=m3db001 m3dbnode -f config.yml
 ```
 
 ### Kernel
-`m3dbnode` uses a lot of mmap-ed files for performance, as a result, you might need to bump `vm.max_map_count`. We suggest setting this value to 262,144 when provisioning your VM, so you don’t have to come back and debug issues later.
-
-### OS
-`m3dbnode` also can use a high number of files and we suggest setting a high max open number of files due to per partition fileset volumes.
-
-On linux you can set a high limit for number of max open files in `/etc/security/limits.conf`:
-```
-your_m3db_user        hard nofile 500000
-your_m3db_user        soft nofile 500000
-```
-
-Alternatively, if you wish to have `m3dbnode` run under `systemd` you can use our [service example](https://github.com/m3db/m3/tree/master/integrations/systemd/m3dbnode.service) which will set sane defaults.
-
-Before running the process make sure the limits are set, if running manually you can raise the limit for the current user with `ulimit -n 500000`.
+[Recommended kernel configuration](../operational_guide/kernel_tuning.md).
```

Comment:
Can you make this a complete sentence that explains why its important that people follow this link. I.E something like: Make sure to review our recommended kernel configuration before running M3DB in production as M3DB may exceed the default limits for some default kernel values

Reply:
Also its called Kernel Configuration here and Kernel Tuning below. Lets pick one (configuration seems more consistent to me)

Reply:
sounds good

Reply:
done